packages/kbn-build/package.json

```diff
@@ -5,36 +5,60 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "build": "webpack"
+    "build": "webpack",
+    "prettier": "prettier --single-quote --write './src/**/*.ts'"
```

Comment:
Hm, I wonder if it's possible to handle this in the ESLint setup, https://github.com/elastic/kibana/blob/master/.eslintrc.js#L17. Maaaybe an additional override that specifies `parser` as https://github.com/eslint/typescript-eslint-parser?

Reply:
I didn't manage to make it work from the first try, but will try again :)